packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -104,7 +104,7 @@ module.exports = (
       options: {
         ...EMAILS_DOCS.RECOVERY_EMAIL_STATUS_GET,
         auth: {
-          strategy: 'sessionToken',
+          strategy: 'sessionTokenNoAssurance',
```

Comment:
well I don't have a good suggestion for a new name but 'NoAssurance' might be confusing to someone who's not familiar with AALs.  Of course we could argue that whoever reading this probably should be familiar with AALs.  🤷

Reply:
By 'confusing' I meant 'misleading'.